Makefile

```diff
@@ -205,6 +205,16 @@ install-kuttl:
 		rm -rf $(KUTTL_TAR_FILE_NAME); \
 	fi
 
+ASSERT_LOCATION ?= /usr/local/bin/kubectl-assert
+ASSERT_BINATY_INSTALLATION ?= "https://raw.githubusercontent.com/morningspace/kubeassert/master/kubectl-assert.sh"
+.PHONY: install-assert
+install-assert:
+	@if ! test -f "$(ASSERT_LOCATION)"; then \
+		curl -L $(ASSERT_BINATY_INSTALLATION) -o kubectl-assert; \
+		chmod +x kubectl-assert; \
+		sudo mv ./kubectl-assert /usr/local/bin; \
```

Comment:
move the binary to $(LOCALBIN) instead of /usr/local/bin. Change it for kuttl as well please